### PR TITLE
reformat and lint javascript code.

### PR DESCRIPTION
**Pull request by dong77**

### CODE REVIEW

**Comment by BenjaminPrice** [comment]
Why the change to the lint rules? (I'm quite fond of the ones I had added)
Also, looks like you need to merge master into your branch.

**Comment by dong77** [comment]
READY to be reviewed @BenjaminPrice

**Comment by BenjaminPrice** [comment]
Will review this evening.

**Comment by dong77** [comment]
The lint need to support `async` which is only available (to my knowledge) with the parser I specified. You don’t have to change your editor, just need to run `./node_modules/.bin/eslint . —fix`

File: .eslintrc.json [hunk 1]
```diff
@@ -1,47 +0,0 @@
-{
-    "parserOptions":
-    {
-        "ecmaVersion": 2017
-    },
-    "plugins": [
-        "jest"
-    ],
-    "rules": {
```
Comment (BenjaminPrice) [inline .eslintrc.json]: I would really prefer we re-add the rules around formatting of curly braces, white spacing, and variables. Do you have any objection to keeping them?

File: .eslintrc.json [hunk 2]
```diff
@@ -1,47 +0,0 @@
...
-        "no-inline-comments": 2,
-        "no-mixed-spaces-and-tabs": 2,
-        "no-multi-spaces": 2,
-        "no-trailing-spaces": 2,
-        "no-undef": 2,
-        "no-unused-vars": [2, {"args": "none"}],
-        "no-var": 1,
-        "no-const-assign": 2,
-        "prefer-const": 2,
-        "semi": [2, "always"],
```
Comment (BenjaminPrice) [inline .eslintrc.json]: I think it's important to keep semi-colons.

**Review by BenjaminPrice — COMMENTED** [review]
If the lint rules are readded this will require an additional review to confirm the formatting.

File: .eslintrc.json (hunk 1 above)
Comment (dong77) [reply .eslintrc.json]: I don't. I'll add them back.

Comment (dong77) [reply .eslintrc.json]: I added them back.

**Review by BenjaminPrice — APPROVED** [review]

**Pull request merged**